lib/reporter/logger.js

```diff
@@ -12,6 +12,7 @@ function logModule(log, logType, module) {
     log[logType](chalk.yellow('version:'), module.version);
   }
   if (module.error) {
+    if (!module.testOutput) module.testOutput = '';
     log[logType]('error:', module.error.message);
```

Comment:
Should it be after this line?

Reply:
Idon't think it makes a difference either way, could also be before it. Open to changing if there is a reason

Reply:
Of course... Somehow I interpreted your line as an early `return`